guide/pitchstaircase2.svg

```diff
@@ -0,0 +1,361 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
funny alignment of some elements (ratios) and the aspect ratio of the text is off.